modules/porous_flow/src/actions/PorousFlowActionBase.C

```diff
@@ -303,6 +332,8 @@ PorousFlowActionBase::addStressAux()
   {
     std::string aux_kernel_type = "RankTwoAux";
     InputParameters params = _factory.getValidParams(aux_kernel_type);
+    if (_subdomain_names_set)
+      params.set<std::vector<SubdomainName>>("block") = _subdomain_names;
```

Comment:
if the variable is block-restricted already this might not be necessary?

Reply:
Oh - I overlooked that *we* create stress_xx.